Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,19 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncode() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        let tmpDirPath = NSTemporaryDirectory()
+        let fileURL = URL(fileURLWithPath: tmpDirPath.appending("testEncodeFile"))
+
+        do {
```

Comment:
Instead of using `do-catch` here, you should use `XCTAssertNoThrow`s.

Reply:
I have 3 calls that have try in them (make temporaryDirectoryURL, encode, removeItem).
Should I only use `XCTAssertNoThrow` for `encode` method but inside do{} block?

Reply:
Or actually, if you want you can keep it like this, just get rid of the `XCTAssert(true)` and instead create other assertions to check:
1. The new file exists
2. It contains the expected JSON value
3. It no longer exists after calling `removeItem`